tools/mafft/macros.xml

```diff
@@ -2,6 +2,11 @@
 <macros>
     <token name="@VERSION@">0</token>
```

Comment:
It would be good to increate the wrapper version there

Reply:
Exactly what I was asking Paul. I was in a dilemma given nothing else changes.

Reply:
Thanks, yes. Done.